ihtcvirtualgipssolution/src/ihtcvirtualgipssolution/Model.gipsl

```diff
@@ -387,9 +406,7 @@ constraint with assignedGenderToRoomOnShift {
 
 // H3: Surgeon overtime: The maximum daily surgery time of a surgeon must not be exceeded. 
 constraint with surgeonOptimeTupel {
-	mappings.selectedOperationDay->filter(
-		element.nodes.vwop.opTime == context.nodes.op
-	)->sum(element.nodes.p.surgeryDuration * element.value)
+	mappings.selectedOperationDay->join((opTime, op))->sum(element.nodes.p.surgeryDuration * element.value)
```

Comment:
Ah, I have a feeling that the adaptations on the patterns were necessary to replace some of the `filter`s (e.g., what I see here) with `join`s.

Please enlighten me :).